habitat-baselines/habitat_baselines/rl/hrl/skills/skill.py

```diff
@@ -189,33 +192,35 @@ def should_terminate(
             dtype=torch.bool,
         )
         if self._config.max_skill_steps > 0:
-            over_max_len = cur_skill_step > self._config.max_skill_steps
+            over_max_len = cur_skill_step >= self._config.max_skill_steps
```

Comment:
If should_terminate runs every step, try not to access the _config but store max_skill_steps in a variable